lib/OnyxConnectionManager.ts

```diff
@@ -227,7 +227,7 @@ class OnyxConnectionManager {
      * Adds the connection to the eviction block list. Connections added to this list can never be evicted.
      * */
     addToEvictionBlockList(connection: Connection): void {
-        const connectionMetadata = this.connectionsMap.get(connection.id);
+        const connectionMetadata = this.connectionsMap.get(connection?.id);
```

Comment:
why not just return early above if `connection` is undefined before trying to access the key?

Reply:
@getusha  We already have a check to return early if `connectionMetadata` so I think we can do this and no need to add a new if condition